src/components/organisms/footer/footer.scss

```
@@ -0,0 +1,121 @@
+@use "../../variables" as *;
+.footer {
+  .footer-top {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    padding: 52px;
+
+    .switch {
+      margin-bottom: 20px;
+
+      .switch-img {
+        width: 150px;
+      }
+    }
+
+    address {
+      text-align: center;
+      font-size: 14px;
+      line-height: 28px;
+      margin-bottom: 20px;
+    }
+
+    .icon {
+      display: flex;
+      gap: 14px;
+
+      a {
+        color: $black;
+        text-decoration: none;
+        transition: 0.2s ease;
+        &:hover {
+          color: $gray_nobel;
+        }
+      }
+    }
+  }
+
+  .link {
+    background-color: $white_smoke;
+    padding-left: 0;
+    display: flex;
+    justify-content: center;
+    margin: 0;
+    padding: 16px;
+    border-top: 1px solid;
```

```suggestion
  .link {
    background-color: $white_smoke;
    padding-left: 0;
    display: flex;
    justify-content: center;
    margin: 0;
    padding: 16px;
    border-top: 1px solid;
    row-gap: 8px;
```
display: flex や grid を設定している場合、要素同士の間隔は `gap` で調整するのが便利です。
（row-gapは縦の間隔のみを設定するプロパティです）
参考：https://developer.mozilla.org/ja/docs/Web/CSS/gap